modules/tensor_mechanics/doc/content/source/bcs/PenaltyInclinedBC.md

```diff
@@ -0,0 +1,21 @@
+# PenaltyInclinedBC
+
+!syntax description /BCs/PenaltyInclinedBC
+
+## Description
+
+`PenaltyInclinedBC` is a `IntegratedBC` used for enforcing inclined boundary conditions $\mathbf{u}\cdot \mathbf{normal} = 0$. With a penalty method, the residual is given as
+\begin{equation}
+\mathcal{R}_i = \alpha(\mathbf{u}\cdot \mathbf{normal})\mathbf{normal}(\text{component})\psi_i
+\end{equation}
+where $\alpha$ is the penalty number and `component` corresponds to the direction in which to apply the residual.
```

Comment:
Clarify here that this is specifically for mechanics problems, and that the normal comes from the surface normal. Also, make sure you specify that you have to provide an instance of this BC for each displacement component. For other BCs like this (such as Pressure), we have an Action that sets up all 3 of those.  We should probably provide the same here.